src/index.html

```diff
@@ -6,12 +6,33 @@
 	<title>Chatosaurus</title>
 </head>
 <body>
-
-<h1>My First Heading</h1>
-
-<div id="container"></div>
-
-<img src="./img/nitro-dude.png" alt="img"/>
-
+	<div class="app">
+		<nav class="side">
+			<div class="side-server">
+				<div class="side-server-home"></div>
+				<div class="side-server-channel"></div>
+				<div class="side-server-download"></div>
+			</div>
+			<div class="side-message">
+				<div class="side-message-start"></div>
+				<div class="side-message-menu"></div>
+				<div class="side-message-convo">
+					<div class="side-message-convo-head">
+						<span>DIRECT MESSAGES</span>
+					</div>
+					<div class="side-message-convo-content"></div>
+				</div>
+				<div class="side-message-options"></div>
+			</div>
+		</nav>
+		<main class="chat">
+			<div class="chat-top">
+				<div class="chat-top-menu"></div>
+				<div class="chat-top-icons"></div>
+			</div>
+			<div class="chat-content"></div>
+			<div class="chat-send"></div>
```

Comment:
What is the intention of these elements? Is there an OOCSS name you could give these items?